kernel/aster-nix/src/syscall/utimens.rs

```diff
@@ -140,6 +140,8 @@ fn vfs_utimes(dentry: &Arc<Dentry>, times: Option<TimeSpecPair>) -> Result<Sysca
     // Update times
     dentry.set_atime(atime);
     dentry.set_mtime(mtime);
+    let now = RealTimeCoarseClock::get().read_time();
```

Comment:
can we return the `ctime` in line 112 ? so there is no need to call `RealTimeCoarseClock::get().read_time()` multiple times.

Reply:
That's better! Thank you for your reminder.